dexbot/strategies/relative_orders.py

```diff
@@ -58,6 +58,7 @@ def __init__(self, *args, **kwargs):
         if self.is_center_price_dynamic:
             self.center_price = None
             self.center_price_depth = self.worker.get('center_price_depth', 0)
+            self.cp_from_last_trade = self.worker.get('cp_from_last_trade', False)
```

Comment:
Change `self.worker.get('cp_from_last_trade', False)` -> `self.worker.get('center_price_from_last_trade', False)` to match the ConfigElement